### PR TITLE
Use cmake in docs and fix Issue 100 - scan cmd

### PR DESCRIPTION
**Pull request by sarah-iot**
* Removes Makefile references
* Resolves: https://github.com/openthread/ot-docs/issues/100

### CODE REVIEW

**Comment by sarah-iot** [comment]
Hi @jwhui - this could use a tech review too; not sure who to add...thanks!

File: site/en/guides/border-router/docker/test-connectivity.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ switches.
 See [Build OpenThread](../../../guides/build.md) for basic building instructions.
 
 See the [Build a Thread network with nRF52840 boards and OpenThread
-Codelab](https://codelabs.developers.google.com/codelabs/openthread-hardware/#0) for 
+Codelab](https://codelabs.developers.google.com/codelabs/openthread-hardware/#0) for
```
Comment (Vyrastas) [inline site/en/guides/border-router/docker/test-connectivity.md:54]: The domain can be openthread.io now

**Review by Vyrastas — APPROVED** [review]
One minor link nit, otherwise LGTM from TW perspective

File: site/en/guides/border-router/docker/test-connectivity.md (hunk 1 above)
Comment (sarah-iot) [reply site/en/guides/border-router/docker/test-connectivity.md]: Thanks - done.

**Review by jwhui — APPROVED** [review]
Looks great. Thanks! 👍🏼

**Pull request merged**